tasks/connectors/edgescan/lib/edgescan_vulnerability.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Kenna
+  module Toolkit
+    module Edgescan
+      class EdgescanVulnerability
+        attr_accessor :asset, :data
+
+        def initialize(asset, vulnerability)
+          @asset = asset
+          @data = vulnerability
+        end
+
+        def specifier_id
+          data["location_specifier_id"]
+        end
+
+        def location
+          data["location"]
+        end
+
+        def matching_location_specifier
+          asset.find_location_specifier(specifier_id, location)
+        end
+
+        def to_kenna_vulnerability
+          {
+            "scanner_type" => "Edgescan",
+            "scanner_identifier" => data["definition_id"],
+            "created_at" => data["created_at"],
+            "last_seen_at" => data["updated_at"],
+            "scanner_score" => data["threat"],
+            "score_map" => { "5" => "10", "4" => "8", "3" => "6", "2" => "4", "1" => "2" },
+            "status" => data["status"],
```

Comment:
sorry I missed this the first time but it looks like all the scanner scores I can see are under 5. score_map isn't a field in the KDI so it is likely just getting ignored. Can you use the map to update the value you have in scanner_score?

Reply:
@edblaz

Reply:
It's great that you caught that! 

The `threat` on Edgescan will always be an integer between 1 and 5. That means that what we're supplying to Kenna will always be between 2 and 10. I hope that works!